api/v1alpha1/nodepool_types.go

```diff
@@ -317,6 +317,55 @@ type NodePoolPlatform struct {
 	Agent *AgentNodePoolPlatform `json:"agent,omitempty"`
 
 	Azure *AzureNodePoolPlatform `json:"azure,omitempty"`
+
+	// PowerVS specifies the configuration used when using IBMCloud PowerVS platform.
+	//
+	// +optional
+	PowerVS *PowerVSNodePoolPlatform `json:"powervs,omitempty"`
+}
+
+// PowerVSNodePoolPlatform specifies the configuration of a NodePool when operating
+// on IBMCloud PowerVS platform.
+type PowerVSNodePoolPlatform struct {
```

Comment:
ServiceInstanceID and Subnet is not going to change in a given cluster, so wondering if I can just rely on the specs from the `PowerVSPlatformSpec` from hostedcluster_types.go in nodecontroller? @enxebre